examples/door-lock-app/door-lock-common/door-lock-app.matter

```diff
@@ -1520,6 +1759,8 @@ endpoint 0 {
   server  cluster GeneralDiagnostics;
   server  cluster LocalizationConfiguration;
   server  cluster NetworkCommissioning;
+  binding cluster OtaSoftwareUpdateProvider;
+  server  cluster OtaSoftwareUpdateRequestor;
   server  cluster OperationalCredentials;
   server  cluster PowerSource;
```

Comment:
Is a door lock a power source?

Reply:
Isn't the power source cluster used to inform the client of the power source status and power source type of the server?

`11.7. Power Source Cluster
This cluster is used to describe the configuration and capabilities of a physical power source that provides power to the Node. In case the Node has multiple power sources, each is described by its own Power Source cluster. The Power Source Configuration cluster referenced by the Root Node device type for the Node provides the overview of the power source(s) of the Node.`

Reply:
The lock-app has power source configured so after the merge of the 2 apps it will be in the zap file